src/dialect/mysql/mysql-driver.ts

```diff
@@ -129,7 +129,7 @@ class MysqlConnection implements DatabaseConnection {
               ? BigInt(insertId)
               : undefined,
           numUpdatedOrDeletedRows:
-            affectedRows !== undefined && insertId !== null
```

Comment:
Oh shit, this has been broken hasn't it 😬

Reply:
I suspect it was. Had a "wtf" moment when I went over it for this PR. 😟